spec/factories/log_entries.rb

```diff
@@ -2,26 +2,22 @@
 #
 # Table name: log_entries
```

Comment:
This table didn't even exist before. At least, not recently. I think it used to exist a long time ago, which was I think how annotate was able to initially create this comment.